### DISCUSSION TITLE
predict_proba for timeseries forecasting

### DISCUSSION THREAD

**Discussion by pj-mathematician** [autogluon/autogluon#2021] [Q&A] [answered]
Hi, I am using this module for my project. I need to find the probability for each of the classes for each timestep but I am not able to find the predict_proba() method for the timeseries predictor class. 
The best model is AutoETS and the sktime AutoETS does have a predict proba functionality. Is there any workaround so that i can generate probabilities here too?
Thanks

**Comment by Innixma** [comment, marked as answer]
Created issue #2023 to track this.
